backend/src/main/java/com/github/clarakraus/susume/blog/BlogController.java

```diff
@@ -19,33 +20,67 @@
 public class BlogController {
     private final BlogService blogService;
 
-    @ResponseStatus(HttpStatus.CREATED)
+   /* @ResponseStatus(HttpStatus.CREATED)
     @PostMapping()
     public void createBlog(@RequestBody Blog blog){
         blogService.createBlog(blog);
     }
-    @GetMapping("/{username}")
-    public ResponseEntity<Blog> getBlogDetails(@PathVariable String username) {
+
+   */
+    @GetMapping("/details")
+    public ResponseEntity<BlogDTO> getBlogDetails(Principal principal) {
         try {
```

Comment:
Diese ganzen set Operationen könnte man gut in einem mapper oder in einem Konstruktor von BlogDTO der einen Blog bekommt verstecken